mockOSApi.v2/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,80 @@
+using mockOSApi.Models;
+using Azure.Security.KeyVault.Secrets;
+
+namespace mockOSApi.Repository;
+
+public interface IUserRepositoryKv
```

Comment:
Ah, so are you using a Key Vault for persisting Users?
That's an interesting approach!

Reply:
yes, using Azure KV for user creds